tests/test_fusion.py

```diff
@@ -0,0 +1,40 @@
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import filecmp
+import os
+import unittest
+
+
+class TestSearch(unittest.TestCase):
+    def setUp(self):
+        self.qruns = ['covidex.t5', 'covidex.sim']
+        self.cmps = ['myfused.txt', 'fused.txt']
+
+        for qrun in self.qruns:
+            os.system(f'wget -q -nc https://ir.nist.gov/covidSubmit/archive/round2/{qrun}')
+
+        os.system('wget -q -nc https://www.dropbox.com/s/scl0lm7x47jrsxh/fused.txt')
```

Comment:
If you're going to use this testing strategy, how about use these?
https://github.com/castorini/anserini/blob/master/docs/experiments-covid.md#round-2

> reciprocal rank fusion(1, 3, 5)